packages/cln_plugin/lib/src/cln_plugin_base.dart

```diff
@@ -1,6 +1,57 @@
 // TODO: Put public facing types in this file.
 
-/// Checks if you are awesome. Spoiler: you are.
-class Awesome {
-  bool get isAwesome => true;
+import 'dart:io';
+
+import 'package:cln_plugin/src/rpcmethods/abstractrpcmethod.dart';
+import 'package:cln_plugin/src/rpcmethods/manifest/getmanifest.dart';
+
+import 'icln_plugin_base.dart';
+
+class Plugin implements ICLNPlugin {
+  // Data members
+  late ManifestMethod manifest;
+  /*
+  Map<String, Object> parameters;
+  bool _parametersReady;
+  JsonConverter converter;
+  CLightningPluginConfig configs;
+   */
+
+  Plugin() {
```

Comment:
```suggestion
  Plugin({bool dynamic = false}) {
```
Considerer to add this during our refactoring inside the Plugin class.

The used need to specify if this is a dynamic plugging or now, and we define that it is not by default because I think it is more safe